client/my-sites/marketing/connections/connections.jsx

```diff
@@ -22,8 +22,8 @@ const SharingConnections = ( { translate } ) => (
 		<QueryKeyringConnections />
 		<QueryKeyringServices />
 		<QueryPublicizeConnections selectedSite />
-		<SharingServicesGroup type="publicize" title={ translate( 'Publicize Your Posts' ) } />
-		<SharingServicesGroup type="other" title={ translate( 'Manage Connections' ) } />
+		<SharingServicesGroup type="publicize" title={ translate( 'Publicize posts' ) } />
+		<SharingServicesGroup type="other" title={ translate( 'Manage connections' ) } />
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.